Packs/Core/Playbooks/playbook-Netcat_Makes_or_Get_Connections.yml

```diff
@@ -0,0 +1,1033 @@
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "39":
+    id: "39"
+    taskid: 1751aebc-28fb-4aa4-847d-f8f0e2c82d1d
+    type: regular
+    task:
+      id: 1751aebc-28fb-4aa4-847d-f8f0e2c82d1d
+      version: -1
+      name: Search related alerts by  MITRE technique
+      description: |+
+        This task searches for suspicious alerts related to an incident by mitre techniques that may indicate a compromised user.
+        Focus on identifying alerts associated with the following MITRE techniques & tactics:
+        - T1059- Command and Scripting Interpreter
+        - T1072 - Software Deployment Tools
+        - TA0010 - Exfiltration
+        - T0006 - Credential Access
+      scriptName: SearchIncidentsV2
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "31"
+    scriptarguments:
+      query:
+        complex:
+          root: alert
+          accessor: parentXDRIncident
+          transformers:
+          - operator: Cut
+            args:
+              delimiter:
+                value:
+                  simple: '-'
+              fields:
+                value:
+                  simple: "2"
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: '(mitreattcktechnique:*T1059* or mitreattcktechnique:*T1072* or mitreattcktactic:*TA0010* or mitreattcktactic:*TA0006*) and caseid:'
+              suffix: {}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1830
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "40":
+    id: "40"
+    taskid: 23878a84-73ba-4164-8964-6122438ca5b0
+    type: regular
+    task:
+      id: 23878a84-73ba-4164-8964-6122438ca5b0
+      version: -1
+      name: Close the Alert as False Positive
+      description: commands.local.cmd.close.inv
+      script: Builtin|||closeInvestigation
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "23"
+    scriptarguments:
+      closeReason:
+        simple: Resolved - Handled by the playbook "Netcat makes or gets connections" as False Positive
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 950,
+          "y": 2505
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "41":
+    id: "41"
+    taskid: 796f08e4-11e5-49c2-86bc-5fcc8f734672
+    type: regular
+    task:
+      id: 796f08e4-11e5-49c2-86bc-5fcc8f734672
+      version: -1
+      name: Terminate Causality (CGO)
```

Comment:
After that command why don't we do like in Omri's and Sasha's playbooks - error path on error + suggest manual termination of process to analyst on failure?
@efelmandar  @melamedbn